### DISCUSSION TITLE
Missing licenses

### DISCUSSION THREAD

**Discussion by coberhauser** [DependencyTrack/dependency-track#2391] [Q&A] [answered]
First of all thank you for this awsesome tool. Helps a lot!

However we are experiencing a lot of policy violations due to missing licenses in the DB. Specifically 
- Android Software Development Kit License [https://developer.android.com/studio/terms](https://developer.android.com/studio/terms)
- Amazon Software License https://aws.amazon.com/asl/](https://aws.amazon.com/asl/%22)

Both don't seem to have a SPDX Id whereas they are listed in [LicenseDB](https://scancode-licensedb.aboutcode.org/index.html).
Our BOMs include these licenses but Dependency Track just ommits them when importing the BOM. 

Is there a possiblity to update the license list or is this fixed to the SPDX licenses? I've seen the release notes for 4.7 and possibility to add custom licences, however would a new BOM import not remove the link to those licenses again?

**Comment by stevespringett** [comment, marked as answer]
Currently, DT supports resolved licenses and unresolved licenses. Resolved means that they tie back to SPDX identifier. Unresolved means they do not. The license policy support only operates on resolved licenses.

There is a ticket open to add support for resolved, non-SPDX licenses such as ones defined in LicenseDB. See #1211
